src/Features/CSharp/Portable/UsePatternMatching/CSharpIsAndCastCheckWithoutNameCodeFixProvider.cs

```diff
@@ -55,8 +55,10 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 getInnermostNodeForTie: true, cancellationToken: cancellationToken);
 
             var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+            var expressionTypeOpt = semanticModel.Compilation.GetTypeByMetadataName("System.Linq.Expressions.Expression`1");
```

Comment:
nit: Is this method available to use here?

https://github.com/dotnet/roslyn/blob/cb400a925dc235b505ee2163dc1501d3433d2a12/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs#L101-L102

Reply:
Should be. Thanks!

Reply:
done.